includes/wikia/nirvana/WikiaApp.class.php

```diff
@@ -433,20 +435,26 @@ public function registerApiController( $className, $path ) {
 	 *
 	 * onArticleViewHeader
 	 *
-	 * This is a hook which serves the needs of registerNamespaceControler
+	 * This is a hook which serves the needs of registerNamespaceController
 	 *
 	 * @param Article $article
 	 * @param bool $outputDone
 	 * @param bool $useParserCache
 	 * @return bool
 	 */
 
-	public function onArticleViewHeader(&$article, &$outputDone, &$useParserCache) {
+	public function onArticleViewHeader( &$article, &$outputDone, &$useParserCache ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$useParserCache". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)